front-end-side/src/Pages/ExpensesPages.js

```diff
@@ -30,10 +39,28 @@ const ExpensesPages = ({ categoryType }) => {
                         {
                             status ? (
                                 info.data.map((item) => {
+                                    const { date, seller, details, amount, category } = item;
+                                    if (category == categoryType)
+                                        return (
+                                            <tr>
+                                                <td>{date}</td>
```

Comment:
I believe it would be smart to slice date (to only show ****-**-**) without T.....

Reply:
Okey, i'm gonna fix